rapids-4-spark-tools/README.md

```diff
@@ -1,78 +1,197 @@
 # Spark Qualification and Profiling tools
 
-The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark is a good fit for those applications.
+The qualification tool is used to rank a set of applications to determine if the RAPIDS Accelerator for Apache Spark 
+is a good fit for those applications. Works with CPU generated event logs.
 
 The profiling tool generates information which can be used for ranking applications debugging and profiling.
-Information such as Spark version, executor information, Properties and so on.  Works with both cpu and gpu generated event logs.
+Information such as Spark version, executor information, properties and so on.  
+Works with both CPU and GPU generated event logs.
 
 (The code is based on Apache Spark 3.1.1 source code, and tested using Spark 3.0.x and 3.1.1 event logs)
 
 ## Prerequisites
-1. Require Spark 3.1.1 or newer installed
+- Spark 3.1.1 or newer installed
+- Java 8 or above
+- Complete Spark event log(s) from Spark 3.0 or above version.
 
-## How to compile
-1. `mvn clean package`
+Optional:
+- maven installed 
+  (only if you want to compile the jar yourself)
+- A running Spark cluster 
+  (only if you choose to run the job on a Spark cluster)
+- hadoop-aws-<version>.jar and aws-java-sdk-<version>.jar 
+  (only if any input event log is from S3)
+  
+## Download the jar or compile it
+You do not need to compile the jar yourself because you can download it from maven repository directly.
+
+Here are 2 options:
+1. Download the jar file from maven repository
```

Comment:
we should update to have a path once we have one

Reply:
Yes that is a TODO once the jar link is identified.